goiscsi_iscsi.go

```diff
@@ -292,13 +324,24 @@ func (iscsi *LinuxISCSI) SetCHAPCredentials(target ISCSITarget, username, passwo
 
 // CreateOrUpdateNode creates new or update existing iSCSI node in iscsid dm
 func (iscsi *LinuxISCSI) CreateOrUpdateNode(target ISCSITarget, options map[string]string) error {
+	err := checkIPAddress(target.Portal)
+	if err != nil {
+		fmt.Printf("\nError invalid portal address %s: %v", target.Portal, err)
+		return err
+	}
+
+	err = checkIQN(target.Target)
+	if err != nil {
+		fmt.Printf("\nError invalid IQN Target %s: %v", target.Target, err)
```

Comment:
Change to log

Reply:
Everywhere `fmt.Print` is being used and hence used the same to maintain consistency, Probably we can add a logger at the later stage.